contrib/roles/linux/kubernetes/tasks/prepare_master.yml

```diff
@@ -202,17 +202,19 @@
 
 - name: Kubernetes Master | Enabling OVN to listen on 6641 and 6642
   shell: |
-    ovn-nbctl set-connection ptcp:6641
-    ovn-sbctl set-connection ptcp:6642
+    ovn-nbctl --timeout={{TIMEOUT}} set-connection ptcp:6641
+    ovn-sbctl --timeout={{TIMEOUT}} set-connection ptcp:6642
   changed_when: false
+  failed_when: false
 
 - name: Kubernetes Master | Setting OVN external_ids
   shell: |
-    ovs-vsctl set Open_vSwitch . external_ids:ovn-remote="tcp:{{ host_public_ip }}:6642" \
+    ovs-vsctl --timeout={{TIMEOUT}} set Open_vSwitch . external_ids:ovn-remote="tcp:{{ host_public_ip }}:6642" \
       external_ids:ovn-nb="tcp:{{ host_public_ip }}:6641" \
       external_ids:ovn-encap-ip="{{ host_internal_ip }}" \
       external_ids:ovn-encap-type="geneve"
   changed_when: false
+  failed_when: false
```

Comment:
This will ignore the error if the command returns an error. Should it be ignored?

Reply:
I'll search in all the scripts where it needs to be updated. This caught my eye since the nbctl/sbctl were hanging in a deployment.

I added to check for failures actually. Isn't it the other way around?

Reply:
It checks for failures by default. If there is a multi-line command it should be chained:
```
ovn-nbctl --timeout={{TIMEOUT}} set-connection ptcp:6641 &&
ovn-sbctl --timeout={{TIMEOUT}} set-connection ptcp:6642
```
Or just two separate blocks for each command.

``failed_when: false`` will make ansible to ignore the error in case it happens.

Reply:
Thanks for the info about the default behavior. I thought I had to add `failed_when` to check for it.

I updated the PR can you PTAL when you have time.

We need to address: `contrib/roles/windows/kubernetes/tasks/setup_ovs_docker.yml` as some point. Maybe @ionutbalutoiu can pick it up if he has free time.